packages/core/src/configuration.ts

```diff
@@ -364,6 +364,15 @@ const radarChart: RadarChartOptions = Tools.merge({}, chart, {
 	}
 } as RadarChartOptions);
 
+/**
+ * options specific to treemap charts
+ */
+const treemapChart: RadarChartOptions = Tools.merge({}, chart, {
```

Comment:
Should this be `const treemapChart: TreeMapChartOptions`? Or is there sufficient overlap between tree and radar?